### ISSUE TITLE
m1 ready?

### ISSUE THREAD

**Issue by mclbdn** [dcangulo/react-native-pkce-challenge#143]
Hi,

is this package m1 Mac ready? I'm struggling with required ndk.

Thank you.

**Comment by mrazekrad** [comment]
Hello, 
i have same problem on MacBook Air with M1. I can install only NDK 24 and above. 
Thx for answer.
Radek

**Comment by dcangulo** [comment]
I don't have an m1 device to test with. 

Can you test https://github.com/dcangulo/react-native-pkce-challenge/pull/144, by using the following instead in your `package.json`:
```
"react-native-pkce-challenge": "https://github.com/dcangulo/react-native-pkce-challenge.git#056736b"
```

**Comment by mrazekrad** [comment]
Hello, thank you for your reaction! 
M1 device working with NDK 24+ (https://github.com/android/ndk/issues/1299#issuecomment-1026338583). After build is error: No version NDK matched the requested version 21.0.6114699 (it is for version: "react-native-pkce-challenge": "^5.1.0". Versions available locally: 24.0.8215888. Is possible change to 24+? Thx

**Comment by dcangulo** [comment]
can you test the branch instead of the released one?

**Comment by mclbdn** [comment]
Hi guys,

the solution proposed by @dcangulo worked! 😍 Works both on AVD and in iOS simulator too.

Thanks @dcangulo

**Comment by mclbdn** [comment]
Closing.

**Comment by dcangulo** [comment]
Released in v5.1.1
